packages/components/src/button/style.scss

```diff
@@ -45,6 +45,7 @@
 		white-space: nowrap;
 		background: $components-color-accent;
 		color: $components-color-accent-inverted;
+		min-width: auto;
```

Comment:
I'm not sure if we should change the `button` component 🤔 --cc @ciampo @mirka

Reply:
Thank you for the ping, @ntsekouras !

The proposed solution is not ideal for a few reasons:

- it affects the `Button` component, which a widely used component. Even a small change like this could introduce regressions for many consumers
- even in case we wanted to apply a fix to the `Button` component, we should not apply it only to the `primary` variant, but the component regardless of the `variant`
the actual reason why the "Edit" button is gheing

Reply:
Thank you @ciampo 🙏  for the detailed guide on this one. 

I have removed the styles added for the widely used `<Button>` component.

Now, I have added a new css class `edit-site-sidebar-navigation-screen__button` for the `<Button>` component used inside `<SidebarNavigationScreen>` component as per your suggestions.

I would like to know your feedbacks on the new implementation, if any.